content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -146,7 +151,7 @@ For example, to download version {{< param "fullversion" >}} on Linux, type:
    sudo apt-get install -y kubectl
    ```
 {{< note >}}
-In releases older than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default.
+In releases older than Debian 12, `/etc/apt/keyrings` does not exist by default.
```

Comment:
Don't remove that line. It's useful to keep it.

Reply:
Hi @sftim , I've checked and actually Ubuntu Jammy (22.04) have this directory:

<img width="764" alt="image" src="https://user-images.githubusercontent.com/1542315/221639008-ef8c9e92-1e50-42c6-bb58-230737c89b89.png">

Does it make sense to keep this note?

Reply:
> In releases **older** than Debian 12 and Ubuntu 22.04, `/etc/apt/keyrings` does not exist by default.

You would need to check Ubuntu 21.10 to see if that statement was wrong (because Ubuntu 21.10 is the most recent Ubuntu release that is **older** than Ubuntu 22.04).
Checking that Ubuntu 22.04 has `/etc/apt/keyrings` is not especially relevant, given the statement. We expect that Ubuntu 22.04 does have a directory named `/etc/apt/keyrings`.

Reply:
@sftim you're right, there's no directory `keyrings` at Ubuntu 21.10. I will rollback this change.